bin/submodule-update.sh

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
```

Comment:
Added this to get past the hardlink error I was getting from have the base image be latest.